napalm_srl/srl.py

```
@@ -2165,7 +2172,7 @@ def _cli_commit(self, message='', revert_in=None):
           output = self.device._jsonrpcRunCli(cmds)
           return self._return_result(output)
       except Exception as e:
-          print("Error occurred in _cli_commit: {}".format(e))
+          logging.error("Error occurred in _cli_commit: {}".format(e))
```

@jbemmel Any reason why not reraising exception here too? It's quite confusing to get no error raised but the config not updated
Same for `compare_config`, in returns None in case of error but that could be also interpreted as no diff.
The `rollback` also seems confusing to me... Basically any function which is not only returning you some data (the `get_` functions) it would make sense not to hide the errors but reraise that so the application can handle that